runtime-extension-model/src/main/java/org/mule/runtime/core/api/extension/MuleExtensionModelDeclarer.java

```diff
@@ -689,7 +684,7 @@ private void declareRoundRobin(ExtensionDeclarer extensionDeclarer) {
     ConstructDeclarer roundRobin = extensionDeclarer.withConstruct("roundRobin")
         .describedAs("Send each message received to the next message processor in a circular list of targets.");
 
-    roundRobin.withRoute("route").withChain();
+    roundRobin.withRoute("route").withMinOccurs(1).withChain();
```

Comment:
isn't this 2? what is the purpose of a round robin with just one route?

Reply:
agree with you, but the component allows it in the XSD.

Reply:
add a comment?